src/commands/misc/taco.js

```diff
@@ -0,0 +1,23 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+
+const fetch = require('node-fetch')
+
+module.exports = class Taco extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'taco'
+    })
+  }
+
+  async run ({ t, author, channel }) {
+    const embed = new SwitchbladeEmbed(author)
+    channel.startTyping()
+    const body = await fetch(
+      'http://taco-randomizer.herokuapp.com/random'
+    ).then(res => res.json())
+    const randomTaco = `${body.base_layer.name} with ${body.mixin.name} garnished with ${body.seasoning.name} and topped of with ${body.condiment.name} with and wrapped in ${body.shell.name}`
```

Comment:
This should be translatable.

Reply:
The hard coded parts of the string are certainly translatable via i18n. The ingredients however would need to be put through a translator. I saw you have some i18n bit in the code, but I'm not sure about a translating API or something. If you can point me in the right direction, I'm happy to fix this.

Reply:
> The hard coded parts of the string are certainly translatable via i18n. The ingredients however would need to be put through a translator. I saw you have some i18n bit in the code, but I'm not sure about a translating API or something. If you can point me in the right direction, I'm happy to fix this.

that actually makes sense, its weird something like "Lettuce com Oion e um pouco de..."

Reply:
So... you want to change anything here?

Reply:
well on their repo theres a .db file so its gonna be kinda hard to get all i think and it should be updated in a while so whats your deal @pedrofracassi